optuna/integration/cma.py

```diff
@@ -442,7 +453,13 @@ def _to_optuna_params(search_space, param_name, cma_param_value):
         if isinstance(dist, IntUniformDistribution):
             r = numpy.round((cma_param_value - dist.low) / dist.step)
             v = r * dist.step + dist.low
-            return v
+            return int(v)
+        if isinstance(dist, IntLogUniformDistribution):
+            exp_value = math.exp(cma_param_value)
+            r = numpy.round((exp_value - dist.low) / dist.step)
+            v = r * dist.step + dist.low
+            return int(v)
```

Comment:
How about limiting the value of `v` to the range of `[low, high]` as in `DiscreteUniformDistribution`?
```suggestion
            return int(min(max(v, dist.low), dist.high))
```

Reply:
Thank you! Applied in https://github.com/optuna/optuna/pull/1302/commits/25f19df463dae7ae04f8989bf7039d60b1bd9a6c.